code/parachain/runtime/picasso/src/lib.rs

```diff
@@ -237,6 +237,8 @@ impl randomness_collective_flip::Config for Runtime {}
 
 parameter_types! {
 	pub NativeAssetId: CurrencyId = CurrencyId::PICA;
+	pub const AssetNameMaxChars: u32 = 32;
```

Comment:
not sure if we need to increase this

Reply:
3060374d5b893267bd2a3ec00e0535a898f81c53